packages/uui-radio/lib/uui-radio.story.ts

```diff
@@ -77,6 +77,65 @@ export const RadioGroup: Story = () =>
       <uui-radio value="2" disabled>Option 2</uui-radio>
       <uui-radio value="3">Option 3</uui-radio>
     </uui-radio-group>
+
+    <uui-radio-group name="f331672b-e6f3-4b73-8b44-67a51a24f296">
+      <uui-radio
+        name="f331672b-e6f3-4b73-8b44-67a51a24f296"
+        id="f331672b-e6f3-4b73-8b44-67a51a24f296_0"
+        data-umb="f331672b-e6f3-4b73-8b44-67a51a24f296_0"
```

Comment:
This property seems to be extraneous on all radio buttons. Could we remove it please? :)

Reply:
Yeh sure it was copy and paste as I am using it in Umbraco Forms hence all the extrra cruft 🙈